gptstonks_api/main.py

```diff
@@ -2,6 +2,7 @@
 import asyncio
 import os
 import uuid
+import json
```

Comment:
We need to run pre-commit before merging to other the imports and format the code.